src/PrestaShopBundle/Resources/config/services/core/grid/grid_position_definition_factory.yml

```diff
@@ -57,3 +57,9 @@ services:
       $positionField: 'position'
       $parentIdField: 'id_category'
       $firstPosition: 1
+
+  PrestaShop\PrestaShop\Core\Grid\Position\PositionDefinition:
```

Comment:
You can't use the class for the service name here, it's used for all the previous service definitions as well, there's no reason one of them is more default than another So you must use service names

Reply:
yes indeed. Reverted